internal/app/mapper/mapper.go

```diff
@@ -5,6 +5,7 @@ import (
 	"regexp"
 	"strings"
 
+	corev2 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
```

Comment:
we wouldn't want to import v2 proto here. The envoy v2/v3 proto usage is localized in transport package.

Reply:
I ended up defining our own `Locality` object and put it in `transport/request.go`. Each version is then responsible for doing the mapping between each version and our definition of locality. We didn't have to do that in the other uses of the mapper because all fields are defined in terms of primitive types.

Let me know if maintaining our own locality object is a reasonable compromise.